### DISCUSSION TITLE
자기 음량 확인을 위한 기능이 있으면 좋겠습니다.

### DISCUSSION THREAD

**Discussion by DolDolDol** [metamong-friends/metamong-friends.github.io#51] [제안하기]
자기 음량 확인을 위한 기능이 있으면 좋겠습니다.

**Comment by ACT-TJ** [comment]
안녕하세요 !!

네. 해당 건 개선 계획엔 포함되어 있으나, 개선 우선순위를 높이는 것으로 조정하였습니다.
심플하게라도 우선 본인의 목소리 확인이 가능하도록 기능 제공할 예정이니 조금만 기다려주세요!!

의견 너무 감사드립니다.!!

**Comment by shlee303** [comment]
@DolDolDol 좌측하단 설정 메뉴에 음량 체크 가능한 playback 기능이 추가됐습니다! 좋은 의견 덕분에 개선했습니다, 감사합니다!
